apis/sgv2-graphqlapi/src/test/java/io/stargate/sgv2/graphql/integration/util/GraphqlIntegrationTest.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright The Stargate Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package io.stargate.sgv2.graphql.integration.util;
+
+import com.google.protobuf.StringValue;
+import io.quarkus.test.common.http.TestHTTPResource;
+import io.stargate.bridge.proto.QueryOuterClass;
+import io.stargate.bridge.proto.QueryOuterClass.Query;
+import io.stargate.bridge.proto.QueryOuterClass.QueryParameters;
+import io.stargate.bridge.proto.StargateBridge;
+import io.stargate.sgv2.api.common.StargateRequestInfo;
+import java.time.Duration;
+import javax.inject.Inject;
+import org.apache.commons.lang3.RandomStringUtils;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+
+public abstract class GraphqlIntegrationTest {
```

Comment:
For now, this relies on inheritance, which is a bit underwhelming. Requests are executed through the bridge.

I would prefer to port `CqlSessionExtension` from the v1 codebase, but it's not yet clear to me if/how a Junit extension can interact with a Quarkus test extension.

Reply:
Yes, we should be able to have the `CqlSessionExtension` here as well, that communicates over the bridge or maybe even directly over the CQL port.. For this to happen, we need to expose `host` and `port` of the target CQL port in the integration tests.. Then you can use Cassandra driver to push needed cql statements.. 

I might be working on this while you are on holiday.. Let's sync once you are back..